src/__tests__/test-utils-wrapper.test.tsx

```diff
@@ -0,0 +1,99 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import fs from 'fs';
+import path from 'path';
+import pluralize from 'pluralize';
+
+import * as components from '../../lib/components';
+import * as dom from '../../lib/components/test-utils/dom';
+import * as selectors from '../../lib/components/test-utils/selectors';
+
+jest.mock('@cloudscape-design/test-utils-core/dom');
+jest.mock('@cloudscape-design/test-utils-core/selectors');
+
+function getComponentsList() {
+  const exceptions: Record<string, string> = {
+    AnnotationContext: 'Annotation',
+  };
+
+  return Object.keys(components)
+    .map(component => exceptions[component] ?? component)
+    .filter(Boolean);
+}
+
+describe('Test utils element wrapper', () => {
+  describe('Snapshot tests', () => {
+    const importPaths = [
+      {
+        type: 'dom',
+        relativePath: '../../lib/components/test-utils/dom/index.js',
+      },
+      {
+        type: 'selectors',
+        relativePath: '../../lib/components/test-utils/selectors/index.js',
+      },
+    ] as const;
+
+    test.each(importPaths)('$type wrapper stays unchanged', ({ relativePath }) => {
```

Comment:
I recommend extracting this test separately from the rest

1. Snapshot tests should be in one group with `documenter.test.ts` and others
2. Functional tests may go into `__tests__/test-utils.test.tsx`

Reply:
Done in https://github.com/cloudscape-design/components/pull/2980